backoffice/src/Views/Email/_IndexBanners.cshtml

```diff
@@ -0,0 +1,15 @@
+<div>
+    @if (TempData.TryGetValue("SuccessMessage", out var successMessage))
```

Comment:
If you moved this to shared and renamed to something like _MessageBanner it could be reused for any success or error messages throughout the site (Forms, Views etc)